### ISSUE TITLE
Dependency on commentjson still required?

### ISSUE THREAD

**Issue by johanneskastl** [kdeldycke/click-extra#1152]
https://github.com/kdeldycke/click-extra/blob/31e548c7dd51c2083c7038041f11a4a8e41367e2/pyproject.toml#L76

While packaging click-extra for openSUSE I found that the commentjson dependency is ugly. It seems to be inactive, which means it still pulls in six for the tests. And it depends on `lark-parser`, which was renamed to lark apparently a while ago.

Could you check if this is still required? Could it be replaced with something maintained?

Kind Regards,
Johannes

**Comment by kdeldycke** [comment]
Yeah, I'm also annoyed by `commentjson` being unmaintained. It's sad as it is a great quality of life improvement, especially for JSON configuration file.

So yeah, if it can make your life as a packager easier, I'll get rid of it. You are doing a great job packaging Click Extra, which is more important as it will increase adoption.

**Comment by kdeldycke** [comment]
Done: Click Extra 4.11.7 has been released without commented JSON support.
